configs/mainnet.yaml

```diff
@@ -135,9 +133,6 @@ MESSAGE_DOMAIN_VALID_SNAPPY: 0x01000000
 SUBNETS_PER_NODE: 2
 # 2**8 (= 64)
 ATTESTATION_SUBNET_COUNT: 64
-ATTESTATION_SUBNET_EXTRA_BITS: 0
-# ceillog2(ATTESTATION_SUBNET_COUNT) + ATTESTATION_SUBNET_EXTRA_BITS
-ATTESTATION_SUBNET_PREFIX_BITS: 6
```

Comment:
Do we want to keep this as config, and update the comment to `ceillog2(ATTESTATION_SUBNET_COUNT)`, so it doesn't need to be computed in `compute_subscribed_subnet`?

Reply:
It's buried here https://github.com/ethereum/consensus-specs/pull/3623